### ISSUE TITLE
Parsing NREL's Synthetic but Realistic Distribution Data Sets (Open DSS)

### ISSUE THREAD

**Issue by alejandroowen** [lanl-ansi/PowerModelsDistribution.jl#311]
Has anyone tried to parse one of the NREL Distribution data sets found here https://www.nrel.gov/grid/smart-ds.html?
They are in Open DSS format but when trying to parse the Greensboro NC data I run into several errors, such as:

*Two-phase loads not supported
*Fuses not supported 
*emerghkva_2 not defined

I have been able to get around the first two, but I am not sure what the last one is since there is no "emerghkva_2" in the data set that I can see. There are, however, "emerghkva" values for each winding of a transformer. This is an example of what can be found in the Greensboro data set:

New Transformer.tr(r:udt29-udt29lv) phases=1 windings=3 %loadloss=0.798816 %Noloadloss=0.472 normhkva=27.5 wdg=1 conn=wye bus=udt29.1 Kv=7.199560000000001 kva=25.0 EmergHKVA=37.5 %r=0.266272 wdg=2 conn=wye bus=udt29lv.1.0 Kv=0.120089 kva=25.0 EmergHKVA=37.5 %r=0.532544 wdg=3 conn=wye bus=udt29lv.0.2 Kv=0.120089 kva=25.0 EmergHKVA=37.5 %r=0.532544 XHL=2.4 XLT=2.4 XHT=1.6

Does anyone know how to get PMD take a Open DSS command like the one above and stop getting an "emerghkva_2" related error?

Any help would be appreciated,

Thanks!

**Comment by pseudocubic** [comment]
I'm pretty sure that last one is a bug. When we parse in transformers that are defined that way, we append `_{wdg#}` during the parse to keep the variables unique while waiting to build the PMD data model. There is clearly a problem somewhere, and I have a feeling that it has to do with not having defined that that field can be specified per winding. I will try to track this down. 

Thanks for the bug report!

**Comment by pseudocubic** [comment]
I believe that this should be working in an upcoming version 0.10, which you can test now using the 0.10-rc branch via `] add PowerModelsDistribution#0.10-rc` in Julia.

Let me know if that solves your problem!

As a side note, as I was trying to track this down, I found that specifying `EmergHKVA` for each winding doesn't do anything in OpenDSS, since it is not a per winding specified property. Instead, what seems to happen is that the final value for that property that appears in the definition string is what gets written to the final component object, e.g. if we had the following transformer definition derived from the example you gave:

```dss
New Transformer.tr(r:udt29-udt29lv) phases=1 windings=3 
~ wdg=1 bus=udt29.1 EmergHKVA=1 
~ wdg=2 bus=udt29lv.1.0 EmergHKVA=2 
~ wdg=3 bus=udt29lv.0.2 EmergHKVA=3
```
then EmergHKVA should be 3 in the final transformer object, and there would _not_ be an EmergHKVA array with `[1,2,3]` in it.

**Comment by alejandroowen** [comment]
I tried this yesterday with the 0.10-rc branch you mentioned and it worked. I was able to parse the dss files. 

On another note. After parsing the data set via `eng = parse_file("..\\Master.dss") ` I tried running an OPF by doing `run_mc_opf(eng, ACPPowerModel, ipopt_solver)` and got the following error. 


![unnamed](https://user-images.githubusercontent.com/69917992/99118964-76187a00-25c6-11eb-8bcf-3f48bfdb6abe.png)


Should I create a separate issue for this or leave it in this threat?

Thanks a lot for the help.
